src/actions/notificationsActions.js

```diff
@@ -163,8 +165,17 @@ export const startListeningNotificationsAction = () => {
       contacts: { data: contacts },
     } = getState();
     if (SOCKET && SOCKET.socket && SOCKET.socket.readyState === 1) {
-      SOCKET.onMessage(async response => {
-        const data = JSON.parse(response.data.msg);
+      SOCKET.onMessage((response) => {
+        let data;
+        try {
+          data = JSON.parse(response.data.msg);
+        } catch (e) {
+          // this shouldn't happen, but was reported to Sentry as issue, let's report with more details
+          Sentry.captureMessage('Platform WebSocket notification parse failed', { extra: { response } });
+          return; // unable to parse data, do not proceed
+        }
+
+        const senderUserId = get(data, 'senderUserData.id');
```

Comment:
What would happen if `senderUserId` is `undefined`?

Reply:
Hm, it has been like this, but will add additional check here 👌 thanks

Reply:
Actually before making more changes here we should ask @alazarevski if it's ok to check whether `senderUserId` is present or those actions need to be called even if it's null / undefined?